content/docs/use-cases/experiment-tracking.md

```diff
@@ -0,0 +1,102 @@
+# Machine Learning Experiment Tracking
+
+Making progress on data science projects requires a large number of
+<abbr>experiments</abbr> -- attempts at tuning parameters, trying different
+data, improving code, collecting new metrics, etc. Keeping track of all these
+changes is challenging but essential, as we may need to review, compare, and
+share them. We also want to be able to recover the conditions that produced
+certain results before, and reproduce them if needed.
+
+![](/img/natural-experimentation.png) _Loose experimentation_
+
+DVC provides a layer of [experiment management] features out-of-the-box (no need
+for special services, servers, or websites). Running **DVC Experiments** in your
+workspace captures all the relevant changes automatically (input data, code,
+hyperparameters, artifacts, etc.) to be reviewed and restored later. This is
+possible because the entire experimental process is [codified] with DVC.
+
+Unlike tools that focus on experiment navigation (see a
+[comparison](#comparison-of-popular-experiment-tracking-tools) below), DVC's
+approach guarantees reproducibility by working on top of Git instead of as a
+separate system with fragile revision links. Experiments tracked by DVC are
+associated with the baseline project versions from which they came.
```

Comment:
This seems to explain Experiment Versioning after all. Is that OK to keep here (also featured in the comparison table) or do we want to avoid the versioning concept completely? WDYT @dberenbaum

Reply:
I think it's a good thing to include it.